.gitignore

```diff
@@ -0,0 +1,5 @@
+.eslintrc.json
+.stylelintrc.json
+package.json
+package-lock.json
```

Comment:
- [ ] Kindly remove these files from the `.gitignore` file as they are essential to this project. the first two are linter config files, and the second two hold your project configurations which are part of the code review process. 
- [ ] I don't know how but the package-lock.json file is on the branch but the rest are missing
![image](https://user-images.githubusercontent.com/81932478/189221723-0adc8b83-e1fe-4551-9465-e1ac22dcf1a6.png)

Reply:
@Munsa1  resolved thanks a lot for mentioning that